huaweicloud/services/rds/resource_huaweicloud_rds_instance.go

```diff
@@ -168,8 +168,13 @@ func ResourceRdsInstance() *schema.Resource {
 						},
 						"keep_days": {
 							Type:     schema.TypeInt,
+							Default:  7,
```

Comment:
according to the API, the default value is 0?

Reply:
![default](https://github.com/huaweicloud/terraform-provider-huaweicloud/assets/121535229/42d8e283-f659-4014-9853-2e32b25855c4)

The default is fine.

Reply:
the resource name is rds_instance not backup_strategy, so we should refer to the console or API of creating an instance